tools/cldr-code/src/main/java/org/unicode/cldr/api/AttributeKey.java

```diff
@@ -92,9 +92,9 @@ public static AttributeKey keyOf(String elementName, String attributeName) {
         // 1) we don't expect the attribute name to have a namespace either,
         // 2) the attribute key should be in our cache of known instances.
         if (elementName.indexOf(':') == -1) {
-            checkArgument(attributeName.indexOf(':') == -1,
-                "attributes in an external namespace cannot be present in elements in the default"
-                    + " namespace: %s:%s",
+            checkArgument((attributeName.startsWith("xml:") || attributeName.indexOf(':') == -1),
```

Comment:
Does this want to be !attributeName.startsWith("xml:") &&

Reply:
@macchiati No, the first (boolean) argument to checkArgument should be true to indicate argument is OK, i.e. it either starts with "xml:" or does not have ':'

Reply:
@macchiati No, the first (boolean) argument to checkArgument should be true to indicate argument is OK, i.e. it either starts with "xml:" or does not have ':'

The message is for if the check fails, i.e. the boolean value is false

Reply:
Ideally, this should be a function and check against a list.